[2024-03-03T03:42:47.359+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: fetch_data_files.get_files_4 scheduled__2017-01-15T00:00:00+00:00 [queued]>
[2024-03-03T03:42:47.377+0000] {taskinstance.py:1979} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: fetch_data_files.get_files_4 scheduled__2017-01-15T00:00:00+00:00 [queued]>
[2024-03-03T03:42:47.378+0000] {taskinstance.py:2193} INFO - Starting attempt 1 of 1
[2024-03-03T03:42:47.404+0000] {taskinstance.py:2214} INFO - Executing <Task(PythonOperator): get_files_4> on 2017-01-15 00:00:00+00:00
[2024-03-03T03:42:47.415+0000] {standard_task_runner.py:60} INFO - Started process 5028 to run task
[2024-03-03T03:42:47.420+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'fetch_data_files', 'get_files_4', 'scheduled__2017-01-15T00:00:00+00:00', '--job-id', '2308', '--raw', '--subdir', 'DAGS_FOLDER/dummy_dag.py', '--cfg-path', '/tmp/tmpohot0qzj']
[2024-03-03T03:42:47.425+0000] {standard_task_runner.py:88} INFO - Job 2308: Subtask get_files_4
[2024-03-03T03:42:47.461+0000] {logging_mixin.py:188} WARNING - /home/***/.local/lib/python3.11/site-packages/***/settings.py:194 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-03-03T03:42:47.556+0000] {task_command.py:423} INFO - Running <TaskInstance: fetch_data_files.get_files_4 scheduled__2017-01-15T00:00:00+00:00 [running]> on host 716a53fa82e1
[2024-03-03T03:42:47.738+0000] {taskinstance.py:2510} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='fetch_data_files' AIRFLOW_CTX_TASK_ID='get_files_4' AIRFLOW_CTX_EXECUTION_DATE='2017-01-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2017-01-15T00:00:00+00:00'
[2024-03-03T03:42:50.447+0000] {python.py:202} INFO - Done. Returned value was: None
[2024-03-03T03:42:50.460+0000] {taskinstance.py:1149} INFO - Marking task as SUCCESS. dag_id=fetch_data_files, task_id=get_files_4, execution_date=20170115T000000, start_date=20240303T034247, end_date=20240303T034250
[2024-03-03T03:42:50.494+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-03-03T03:42:50.515+0000] {taskinstance.py:3309} INFO - 0 downstream tasks scheduled from follow-on schedule check
